### PR TITLE
[EAGLE-924]urls to healthy and unhealthy region-servers in hbase dashboard is not accurate enough

### PR DESCRIPTION
**Pull request by chitin**
[EAGLE-924]urls to healthy and unhealthy region-servers in hbase dashboard is not accurate enough

- Add status filter in web url,append "?status=healthy" or "?status=unhealthy" to the url
- Fix exception when we change time interval
- Fix dashboard show when no data and exception.

https://issues.apache.org/jira/browse/EAGLE-924

### CODE REVIEW

**Comment by haoch** [comment]
@zombieJ please help review about it.

**Comment by haoch** [comment]
@anyway1021 please confirm about the changes and deploy to QA to validate whether https://issues.apache.org/jira/browse/EAGLE-924 is resolved.

**Comment by chitin** [comment]
@anyway1021 has the problem about [EAGLE-924](https://issues.apache.org/jira/browse/EAGLE-924) been solved? Thanks.

File: eagle-metric/eagle-hbase-web/src/main/webapp/app/apps/hbase/ctrls/overview.js [hunk 1]
```diff
@@ -59,7 +59,10 @@
 
 				$scope.site = $wrapState.param.siteId;
 				var result = cache[name] || activeMasterInfo._promise.then(function (res) {
-						var hostname = cache[hostname] = cache[hostname] || res[0].tags.hostname;
+						if(typeof res[0].tags === 'undefined') {
```
Comment (haoch) [inline eagle-metric/eagle-hbase-web/src/main/webapp/app/apps/hbase/ctrls/overview.js]: Maybe it possible:  `res.length == 0`?

Comment (chitin) [inline eagle-metric/eagle-hbase-web/src/main/webapp/app/apps/hbase/ctrls/overview.js]: Thanks for your review and suggestion, I have fix it.

**Comment by haoch** [comment]
LGTM

**Pull request closed without merging**